crates/pallet-erc20-support/src/lib.rs

```diff
@@ -65,6 +70,17 @@ pub mod pallet {
         /// The currency to be exposed as ERC20 token.
         type Currency: Currency<AccountIdOf<Self, I>>;
 
+        /// Allowance balance type.
+        type AllowanceBalance: From<BalanceOf<Self, I>>
```

Comment:
```suggestion
        type Allowance: From<BalanceOf<Self, I>>
```

The allowance does not represent a balance in the sense this is not the balance of the operations. It is more like a credit.